Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -1278,14 +1278,11 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
     model.activeFramebuffer = newActiveFramebuffer;
   };
 
-  const superSetSize = publicAPI.setSize;
-  publicAPI.setSize = (width, height) => {
-    const modified = superSetSize(width, height);
-    if (modified) {
-      publicAPI.invokeWindowResizeEvent({ width, height });
-    }
-    return modified;
-  };
+  model._onSizeChanged = (_publicAPI, _model, newValue) =>
+    publicAPI.invokeWindowResizeEvent({
+      width: newValue[0],
+      height: newValue[1],
+    });
```

Comment:
@finetjul I was going to fix the conflicts and realized that `model._onSizeChanged` is not wired up anywhere. Is there a pathway for invoking the window resize event now that `publicAPI.setSize()` no longer directly invokes the event?

Reply:
I'm partially reverting this such that setSize directly invokes the window resize event in both the OpenGL and WebGPU RenderWindows. The changes are all internal, so if you want to continue with the idea you had for _onSizeChanged, you can do that without a breaking change.

Reply:
`_onSizeChanged` was being called since this following commit 
https://github.com/Kitware/vtk-js/commit/63787d550a76fc9f27a6be074a76d08fcefad639 

TBH, I do not remember anything from this current PR though...

Reply:
Ah, that's good to know. I forgot about that change. We should document it somewhere.